x-pack/plugins/security_solution/public/resolver/view/panels/process_details.tsx

```diff
@@ -161,7 +162,7 @@ export const ProcessDetails = memo(function ProcessDetails({
       <StyledBreadcrumbs breadcrumbs={crumbs} />
       <EuiSpacer size="l" />
       <EuiTitle size="xs">
-        <h4 aria-describedby={titleId}>
+        <h4 data-test-subj="resolver:node-detail:title" aria-describedby={titleId}>
```

Comment:
:grey_question: I don't think that's equivalent to what we're (trying?) to communicate with the cube, is it?

Reply:
the describedby thing ya mean?

Reply:
I put the `desc` back and added some basic tests for them